torch/_subclasses/fake_tensor.py

```diff
@@ -305,6 +308,13 @@ def from_tensor(t, fake_mode):
     def __repr__(self):
         return f"FakeTensor({self.fake_device}, {self.size()}, {self.dtype})"
 
+    def stride(self):
```

Comment:
hmmm, as much as I don't like branching on modes, should we gate the `symbolic_shapes.create_contiguous(self.shape)` on there being symbolic dims in the tensor ? 

this is going to make all of the stride information incorrect

Reply:
I think this is resolved with latest changes?